README.md

```diff
@@ -1 +1,62 @@
-# fp_lap2_pairs_assignment
+# Telegraph Clone
+
+Pre-Requisites
+=======================
+- Have docker installed on your machine
+- http-server installed globally to run the client side
+
+Installation
+=======================
+
+1. Clone the repository
+2. cd into the repo folder
+3. Run `docker compose up`
+4. Wait for all docker containers to be ready
+
+Usage
+=======================
+
+Client Side
+1. cd into the client folder
+2. run http-server
+3. Open a browser 
+4. Navigate to http://localhost:8080
+5. Complete the form
+6. Click Publish
+
+You are then presented with the post you just made
+
+Available APIs
+#### GET
+- `/` - Greeting message
+- `/{id}` - Retrieve a single post by id
+
+id is composed of title-day-month
+
+#### POST
+- `/` - Create a new post
+    This requires a json payload in the following format...
+    ```{
+        "title": "",
+        "author": "",
+        "content": ""
+    }```
```

Comment:
putting the triple back ticks on their own lines will fix  your formatting bug here